include/bbp/sonata/nodes.h

```diff
@@ -53,6 +53,10 @@ class SONATA_API NodePopulation: public Population
     template <typename T>
     Selection matchAttributeValues(const std::string& attribute,
                                    const std::vector<T>& values) const;
+
+
+    // XXX: need to do this better
+    Selection regexMatch(const std::string& name, const std::string& re) const;
```

Comment:
We would never have a string attribute for edges?

Reply:
Currently not supported, correct.  If the need arises, we can add it later, but it's going to be an expensive query.